apps/agent/README.md

```diff
@@ -0,0 +1,254 @@
+# Agent
+
+Agent is an Aragon app that has the ability to perform arbitrary calls to external contracts. It can be thought of as the external interface of a DAO. See our [guide](https://hack.aragon.org/docs/guides-use-agent) for concrete examples on how to use the Agent app.
```

Comment:
We may want to also note that the Agent can hold assets, like ETH, ERC20 tokens, and ERC712 NFTs.

Reply:
Good idea! It's indeed a major feature.